flaml/nlp/huggingface/training_args.py

```diff
@@ -30,8 +30,9 @@ class TrainingArgumentsForAuto(TrainingArguments):
         ckpt_per_epoch (int, optional, defaults to 1): An integer, the number of checkpoints per epoch.
         per_device_eval_batch_size (int, optional, defaults to 1): An integer, the per gpu evaluation batch size.
         label_list (List[str], optional, defaults to None): A list of string, the string list of the label names.
-            When the task is sequence labeling/token classification, need to set the label_list (e.g., B-PER, I-PER, B-LOC)
-            to obtain the correct evaluation metric. See the example in test/nlp/test_autohf_tokenclassification.py.
+            When the task is sequence labeling/token classification, there are two ways to pass the labels:
+            (1) Pass the token labels; (2) Pass the id. For (2), need to set the label_list which is the list of string labels (e.g., [B-PER, I-PER, B-LOC]).
```

Comment:
"pass the labels" and "pass the token labels" are still repeating phrases without clear meaning. Please paraphrase and make it more clear.

Reply:
tried to rephrase.